docs/kyma/05-04-component-source-url.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Install components from user-defined URLs
+type: Configuration
+---
+
```

Comment:
Please describe what will happen if : 
- I specify remote component which is at the same time a component compiled in kyma-installer like istio
- I specify component like above and fetching the remote chart will fail, will the compiled in be used?
- Describe the retry mechanism, does it differ to the current one?
- Would be nice to describe basic authentication for HTTP. See here https://github.com/hashicorp/go-getter#basicauthentication for GIT we can skip as it requires the key to be present in the container which cannot work.